packages/babel-preset/package.json

```diff
@@ -26,25 +26,25 @@
     "access": "public"
   },
   "dependencies": {
-    "@babel/plugin-proposal-export-default-from": "^7.12.1",
-    "@babel/plugin-proposal-export-namespace-from": "^7.12.1",
+    "@babel/plugin-proposal-export-default-from": "^7.12.13",
+    "@babel/plugin-proposal-export-namespace-from": "^7.14.2",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.12.1",
-    "@babel/preset-env": "^7.12.7",
-    "@babel/preset-react": "^7.12.7",
+    "@babel/plugin-transform-runtime": "^7.14.3",
+    "@babel/preset-env": "^7.14.4",
+    "@babel/preset-react": "^7.13.13",
     "babel-plugin-dev-expression": "^0.2.2",
-    "babel-plugin-polyfill-corejs3": "^0.0.7",
-    "babel-plugin-react-intl": "^7.9.4",
-    "browserslist": "^4.15.0",
-    "lodash": "^4.17.15",
+    "babel-plugin-polyfill-corejs3": "^0.2.2",
+    "babel-plugin-react-intl": "^8.2.25",
```

Comment:
probably should have made this a major change since this has pretty strong effects down the chain

Reply:
grr i didn't mean to bump this